Add support for Symfony 8

loic425 opened the pull request.

loic425 committed:
Add support for Symfony 8

dbu commented:
thanks. i fixed static analysis in #228 and also added tests with php 8.4 and 8.5. can you please rebase (and solve the conflicts in the github action definition)?

loic425 commented:
> thanks. i fixed static analysis in #228 and also added tests with php 8.4 and 8.5. can you please rebase (and solve the conflicts in the github action definition)?

Done :)

dbu commented:
thanks a lot for the contribution! i wrapped things up in #229 and merged, will tag the release now.
